src/pvrclient-nextpvr.h

```diff
@@ -108,8 +108,9 @@ class ATTRIBUTE_HIDDEN cPVRClientNextPVR : public kodi::addon::CInstancePVRClien
   MenuHook& m_menuhook = MenuHook::GetInstance();
   Recordings& m_recordings = Recordings::GetInstance();
   Timers& m_timers = Timers::GetInstance();
-  int64_t m_lastRecordingUpdateTime;
-  time_t m_nextServerCheck = 0;
+  time_t m_lastRecordingUpdateTime;
+  time_t m_nextServerCheck{ 0 };
```

Comment:
Do you want this field to be set to zero or now?

Reply:
Technically I would want this to be zero but  it is only checked  when it is set 60 seconds after  a connection failure.  It is basically for Kodi style.  Does this set it to now?

Reply:
That’s what I’m not sure about. As it’s based  on a basic type I would just always use `= 0;`

Reply:
Checked with the debugger and it is 0 which makes sense.  I changed all time initializations to = 0 for the next release